### PR TITLE
CLI: add -version flag

### LINKED ISSUE #68: Provide -version output
Common best practices for any CLI command.

```
[root@icinga2-prometheus yum.repos.d]# icingadb -version
flag provided but not defined: -version
Usage of icingadb:
  -config string
    	path to config (default "icingadb.ini")
```

### PR DESCRIPTION
**Pull request by Al2Klimov**
fixes #68

### CODE REVIEW

**Comment by Al2Klimov** [comment]
Not like Icinga 2 for machine-readability.

File: main.go [hunk 1]
```diff
@@ -67,8 +68,14 @@ import (
 
 func main() {
 	configPath := flag.String("config", "icingadb.ini", "path to config")
+	versn := flag.Bool("version", false, "print version and exit")
```
Comment (N-o-X) [inline main.go]: Couldn't this be named something like `versionFlag` or `showVersion`?

File: version.go [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package main
+
+const version = "v0.0"
```
Comment (dnsmichi) [inline version.go]: I'd suggest using `1.0.0` similar to how Icinga 2 does it in its VERSION file.

Comment (Al2Klimov) [reply version.go]: 1. This one intentionally complies to `git describe` for packaging.
2. There's no 1.0, yet.

**Comment by Al2Klimov** [comment]
```
➜  icingadb git:(feature/version-68) ✗ go run . -version
Icinga DB version:v2.11.0 build:f14c0ca89
➜  icingadb git:(feature/version-68) ✗ git co .
1 Pfad vom Index aktualisiert
➜  icingadb git:(feature/version-68) ✗ go run . -version
Icinga DB version: v0.0
➜  icingadb git:(feature/version-68) ✗
```

File: version.go (hunk 2 above)
Comment (N-o-X) [inline version.go]: Please use v0.0.0 here

**Pull request merged**